script/testing/junit/BuiltinUDFTest.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Insert statement tests.
+ */
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+public class BuiltinUDFTest extends TestUtility {
+    private Connection conn;
+    private ResultSet rs;
+    private String S_SQL = "SELECT * FROM tbl;";
+
+    private static final String SQL_DROP_TABLE =
+            "DROP TABLE IF EXISTS tbl;";
+
+    private static final String SQL_CREATE_TABLE =
+            "CREATE TABLE tbl (" +
+                    "theta DECIMAL(5,2));";
+
+    /**
+     * Initialize the database and table for testing
+     */
+    private void initDatabase() throws SQLException {
+        Statement stmt = conn.createStatement();
+        stmt.execute(SQL_DROP_TABLE);
+        stmt.execute(SQL_CREATE_TABLE);
+    }
+
+    /**
+     * Get columns, for follow on value checking
+     */
+    private void getResults() throws SQLException {
```

Comment:
I may be wrong But I dont see this being used anywhere. Is it required, When you are checking you are doing select function() from table , right?